### ISSUE TITLE
stripchat.py

### ISSUE THREAD

**Issue by norogodod** [Damianonymous/streamlink-plugins#25]
Hi!
During this last week the plugin for stripchat has been unable to open streams.

I belive the issue is in that the source seem to have changed.
Here are two examples of sources that streams use nowadays.
https://b-hls-05.doppiocdn.com/hls/57297042/57297042.m3u8
https://b-hls-05.doppiocdn.com/hls/72195463/72195463.m3u8

The existing in the script.
server = "https://b-{0}.strpst.com/hls/{1}/master_{1}.m3u8".format(data["cam"]["viewServers"]["flashphoner-hls"],data["cam"]["streamName"])

server0 = "https://b-{0}.strpst.com/hls/{1}/{1}.m3u8".format(data["cam"]["viewServers"]["flashphoner-hls"],data["cam"]["streamName"])

Hopefully, you'll have the time to check in to the matter ;) 
Cause my own lack of python skills won't figure this out XD

Sincerely,
noro

**Comment by hurdurhurdur** [comment]
You don't need python skills to figure it out. Just replace `strpst.com` with `doppiocdn.com`.
How did you get m3u8 links? Usual 'Developer Tools > Network' couldn't find any.

**Comment by Damianonymous** [comment]
@norogodod Changing the server should be enough.
@hurdurhurdur  Developer tool, finds links.

**Comment by norogodod** [comment]
Hahah, lol! 
Thank you both! @Damianonymous & @hurdurhurdur 

I thought i tried to switch out the links before, but I must have made a serious misstake XD 
Thank you for the help!

I found the links through the developer tools the first time around. But when I was going back to get them for the second time around, I was not able to find them. So I used the info gathered from VLC when using another software instead. However to use the other software to full extent, they have put down quite a hefty subscription on it.
